### DISCUSSION TITLE
v0.24.2

### DISCUSSION THREAD

**Discussion by eginhard** [idiap/coqui-ai-TTS#96] [Versions & Releases]
### Highlights

* Prebuilt wheels now also provided for Mac and Windows
* Hindi fully supported for XTTS fine-tuning

### Features
* Add multilingual phoneme cleaner by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/44
* Support hindi in XTTS tokenizer by @akshatrocky in https://github.com/idiap/coqui-ai-TTS/pull/64
* Normalize unicode characters in text cleaners by @shavit in https://github.com/idiap/coqui-ai-TTS/pull/85

### Fixes
* Improve CLI handling of boolean arguments by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/38
* Use the specified device for pretrained speaker encoder by @ChristianRomberg in https://github.com/idiap/coqui-ai-TTS/pull/45
* XTTS streaming fixes by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/46 and by @gravityrail in https://github.com/idiap/coqui-ai-TTS/pull/59
* Numpy 2 support by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/47 https://github.com/idiap/coqui-ai-TTS/pull/56 https://github.com/idiap/coqui-ai-TTS/pull/81
* VC-related refactors and fixes by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/49
* Update to coqui-tts-trainer 0.1.4 by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/51
* Fix typo in xtts.py by @abrahammathews2000 in https://github.com/idiap/coqui-ai-TTS/pull/53
* build: move umap-learn into optional notebook dependencies by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/50
* fix(xtts): load tokenizer file based on config as last resort by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/57
* Skip audio files that can't be decoded by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/66
* Fix wrong references in documentation by @hykilpikonna in https://github.com/idiap/coqui-ai-TTS/pull/74
* fix(build): restrict spacy version to unbreak installation by @KoljaB in https://github.com/idiap/coqui-ai-TTS/pull/92
* ci(release): switch to cibuildwheel by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/95

## New Contributors
* @ChristianRomberg made their first contribution in https://github.com/idiap/coqui-ai-TTS/pull/45
* @abrahammathews2000 made their first contribution in https://github.com/idiap/coqui-ai-TTS/pull/53
* @gravityrail made their first contribution in https://github.com/idiap/coqui-ai-TTS/pull/59
* @akshatrocky made their first contribution in https://github.com/idiap/coqui-ai-TTS/pull/64
* @hykilpikonna made their first contribution in https://github.com/idiap/coqui-ai-TTS/pull/74
* @shavit made their first contribution in https://github.com/idiap/coqui-ai-TTS/pull/85
* @KoljaB made their first contribution in https://github.com/idiap/coqui-ai-TTS/pull/92

**Full Changelog**: https://github.com/idiap/coqui-ai-TTS/compare/v0.24.1...v0.24.2

<hr /><em>This discussion was created from the release <a href='https://github.com/idiap/coqui-ai-TTS/releases/tag/v0.24.2'>v0.24.2</a>.</em>
